projects/core/src/configurator/commons/facade/configurator-groups.service.ts

```diff
@@ -309,11 +174,22 @@ export class ConfiguratorGroupsService {
     );
   }
 
-  setCurrentGroup(owner: GenericConfigurator.Owner, groupId: string) {
+  /**
+   * Determine the current group.
```

Comment:
Sets current group (if nothing has been visited so far, the first group is chosen)